frontend/src/wizards/Governance/Policy/PolicyWizard.tsx

```diff
@@ -392,6 +403,77 @@ export function PolicyWizardTemplates(props: { policies: IResource[] }) {
           />
         </WizHidden>
 
+        {/* OperatorPolicy */}
+        <WizHidden hidden={(template: any) => template?.objectDefinition?.kind !== 'OperatorPolicy'}>
+          <div>
+            <Title headingLevel="h6">{t('Operator Policy')}</Title>
+            <Text component="small">{t('A operator policy creates operators on managed clusters.')}</Text>
```

Comment:
```suggestion
            <Text component="small">{t('An Operator Policy creates operators on managed clusters.')}</Text>
```

Reply:
@JustinKuli can you explain why the P should be capital? Is it displayed this way? Based on the IBM Style Guide, we need to use sentence-case

Reply:
I was thinking "Operator Policy" is a proper noun. The resource kind is `OperatorPolicy`, so this seemed to fit. Personally I'd be fine with "operator policy", "Operator Policy" or "OperatorPolicy"... but "Operator policy" would be strange to me.

Reply:
So should I change to Capital again?